pyproject.toml

```diff
@@ -42,7 +42,7 @@ all = [
 ]
 
 schema = [
-    "aind-data-schema>=1.0.0,<2.0",
+    "aind-data-schema==1.1.1",
```

Comment:
We should keep a loose pin of `>=1.1.0<2.0` and write a way to ignore schema versions in the unit tests.